bin/manage.py

```diff
@@ -288,6 +294,7 @@ def reload(self):
 # ---------------------------------------------------------
 # Top-level functions called by ContainerPilot or forked by this program
 
+@debug
```

Comment:
Yah. At the top of the file you'll see I did a little expanding of the information in the debug log so that interleaved events can be separated. While doing that it became obvious we needed to tag a couple more events to see the workflow properly.